optimum_benchmark/backends/base.py

```diff
@@ -1,55 +1,60 @@
-from typing import Any, Callable, Dict, List, Optional, Union
-from dataclasses import dataclass, MISSING
+from typing import Any, ClassVar, Dict, List, Optional, Union, TYPE_CHECKING
 from multiprocessing import Process
 from abc import abstractmethod, ABC
+from dataclasses import dataclass
 from logging import getLogger
-import shutil
 import os
 import gc
 
 
-import torch
-from torch import Tensor
-from datasets import Dataset
+import shutil
 from psutil import cpu_count
-from omegaconf import DictConfig
+from diffusers import DiffusionPipeline
```

Comment:
Does this make diffusers a hard requirement of optimum-benchmark?

Reply:
I'll use `transformers` pipeline instead